security/subsystem/src/main/java/org/jboss/as/security/LegacySupport.java

```diff
@@ -328,6 +328,7 @@ private ModelNode getNoTextValueTypeDescription(final ModelNode parent) {
         static {
             final ParametersValidator delegate = new ParametersValidator();
             delegate.registerValidator(CODE, new StringLengthValidator(1));
+            delegate.registerValidator(Constants.MODULE, new StringLengthValidator(1, true));
```

Comment:
why? Whole LegacySupport class is there to provide backward compatibility for older version of the security subsystem
new stuff doesn't need that.